examples/example-spec.yaml

```diff
@@ -49,6 +49,25 @@
   expectStatus: 200
 
 - name: test 4
+  url: http://your-url.com/other/path
+  requestMethod: GET
+  expectData:
+    # In the `contains` case of data validation, a list of matchers is specified. Currently,
```

Comment:
This comment would be good to update to describe the behavior of your new `notContains` case, and you can probably remove the duplicate `keyValueMatch` comments below.

Reply:
Updated!